src/pages/public_registration/action_accordions/PublishingAccordion.tsx

```diff
@@ -193,6 +193,10 @@ export const PublishingAccordion = ({
   const unpublishedOrDeleted =
     registration.status === RegistrationStatus.Deleted || registration.status === RegistrationStatus.Unpublished;
 
+  const filesAwaitingApproval = registration.associatedArtifacts.filter(
+    (artifact) => artifact.type === 'UnpublishedFile'
```

Comment:
kan forenkles ved å gjenbruke lignede operasjon som gjøres for `hasUnpublishedFiles`